stl/inc/ostream

```diff
@@ -493,8 +493,10 @@ public:
                 _TRY_BEGIN
                 _Meta = _Traits::eq_int_type(_Traits::eof(), _Meta) ? _Strbuf->sgetc() : _Strbuf->snextc();
                 _CATCH_ALL
-                _Myios::setstate(ios_base::failbit);
-                _RERAISE;
+                // N4971 [ostream.inserters]/8-9: "...If an exception was thrown

```

Comment:
Reducing this citation from the longer one in #4322 is good, but now this is quoting part of paragraph 9 only, so it should be adjusted accordingly.